src/components/topBar/AboutDialog.tsx

```diff
@@ -275,14 +274,14 @@ function Module({ type, name, version, gitTag }: GridSuiteModule) {
 
 export function AboutDialog({
     open,
-    onClose,
-    globalVersionPromise,
     appName,
     appVersion,
     appGitTag,
     appLicense,
-    additionalModulesPromise,
     logo,
+    onClose,
+    globalVersionPromise,
+    additionalModulesPromise,
```

Comment:
why move these lines, the file isn't changed.

Reply:
done